src/index.ts

```diff
@@ -1,6 +1,27 @@
 import './lib/setup/start';
+
 import { BirthdayyClient } from './lib/BirthdayyClient';
+import { container } from '@sapphire/framework';
+import * as Sentry from '@sentry/node';
+import { SENTRY_OPTIONS } from './config';
+import { SENTRY_DSN } from './helpers/provide/environment';
 
 const client = new BirthdayyClient();
 
-client.login();
+async function main() {
+	try {
+		// Initialize Sentry
+		SENTRY_DSN ?? Sentry.init(SENTRY_OPTIONS);
```

Comment:
this statement makes no sense or am I missing something.
![grafik](https://user-images.githubusercontent.com/58025175/226200714-019f4b04-b168-44f6-a913-13f745ee5fb3.png)
the init gets called with the options when `SENTRY_DSN` is nullish.
But in the options, you define the `dsn` as exactly the `SENTRY_DSN`. 

So you say: `if SENTRY_DSN is nullish execute init with the sentry options that have the dsn included?`

Reply:
check => https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing

Reply:
oh my bad..

Reply:
Oh my bad...